v3-docs/docs/cli/index.md

```diff
@@ -217,210 +217,123 @@ METEOR_CLIENT_ENTRYPOINT=client/main.js METEOR_SERVER_ENTRYPOINT=server/main.js
 You can pass any option that works with `meteor run` to customize the profiling process. This allows you to profile your application under specific conditions that match your deployment environment.
 :::
 
-## meteor create  _app-name_ {meteorcreate}
+## meteor create _app-name_ {#meteorcreate}
 
-The command `meteor create app-name` is the default command for creating a new Meteor project. It creates a subdirectory
-named `app-name` and copies a template app into it. You can pass an absolute or relative path. If you pass a relative
-path, it will be resolved relative to the current working directory. By default, it generates a React project.
+Create a new Meteor project in a directory called `app-name`.
 
```

Comment:
we could mention that `meteor create` calls a wizard to help them choose their app too